voidcrew/_DEFINES/cloning_defines.dm

```diff
@@ -0,0 +1,10 @@
+/// New or not new clone
+#define CLONER_FRESH_CLONE "fresh"
```

Comment:
This define is never used. We check for it, but it's never set by anything ever. We probably don't need the other define too, or the var that uses it. Just hardcode it to work as if it's always using the second define.

Reply:
I will check it out and do that